Resources/Prototypes/Radium/Objectives/groups.yml

```
@@ -0,0 +1,36 @@
+    VoxAmePartFlatpackStealObjective: 1
+    VoxCargoShuttleCircuitboardStealObjective: 1
+    VoxClothingNeckGoldmedalStealObjective: 1              #other
+
+- type: weightedRandom
+  id: VoxThiefObjectiveGroupStructure
+  weights:
+    VoxNuclearBombStealObjective: 0.5
+    VoxFaxMachineCaptainStealObjective: 1
+    VoxChemDispenserStealObjective: 1
+    VoxXenoArtifactStealObjective: 1
+    VoxTegStealObjective: 1
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)